Widgets/WidgetBase/VarmintWidgetRendering.cs

```diff
@@ -33,30 +33,32 @@ public void AdvanceAnimations(GameTime gameTime)
         //--------------------------------------------------------------------------------------
         public virtual void RenderMe(GameTime gameTime)
         {
-            if(Renderer == null)
+            if (Renderer == null)
             {
-                throw new InvalidOperationException("The Renderer property is null on " + this.GetType().Name + " with Name=" + Name);
+                // throw new InvalidOperationException("The Renderer property is null on " + this.GetType().Name + " with Name=" + Name);
             }
+            else
+            {
+                Update();
+                if (!IsVisible) return;
 
-            Update();
-            if (!IsVisible) return;
-
-            bool shouldClip = ClipToBounds
-                || Rotate != 0 || FlipHorizontal || FlipVertical;
-            if (shouldClip) Renderer.BeginClipping(AbsoluteOffset, Size);
-            OnRender?.Invoke(gameTime, this);
+                bool shouldClip = ClipToBounds
+                    || Rotate != 0 || FlipHorizontal || FlipVertical;
+                if (shouldClip) Renderer.BeginClipping(AbsoluteOffset, Size);
+                OnRender?.Invoke(gameTime, this);
 
-            RenderChildren(gameTime);
+                RenderChildren(gameTime);
 
-            if (shouldClip)
-            {
-                float rotation = 0;
-                Renderer.EndClipping(
-                    (float)(Rotate / 180.0 * Math.PI), 
-                    new Vector2(.5f),
-                    new Vector2(1),
-                    FlipHorizontal,
-                    FlipVertical);
+                if (shouldClip)
```

Comment:
It's curious that these lines should show up as changed.  Did you to the CRLF fix that Hans posted to the dev group last week?